src/App.js

```diff
@@ -0,0 +1,22 @@
+import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import BookList from './components/BookList';
+import Categories from './components/Categories';
+import Navigation from './components/Navigation';
+
+function App() {
+  return (
+    <div className="App">
+      <Router>
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<BookList />} />
```

Comment:
- [ ] Kindy edit your route to be like this ` <Route exact path="/">` In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page.